src/Components/Login.js

```diff
@@ -1,30 +1,139 @@
+                    }
+                })
+                .catch((error) => {
+                    console.error(error);
+                    // Show an error message to the user
+                });
+        }
+        else if(selectedOption=="1")
+        {
+            fetch('http://localhost:3000/api/v1/dealers/login', {
+                method: 'POST',
+                headers: {
+                    'Content-Type': 'application/json',
+                },
+                body: JSON.stringify({ Email, Password }),
+                })
+                .then((response) => {
+                    if (response.ok) {
+                        console.log(response.bool);
+                        SetLoggedin(true);
+                        navigate("/dealerpanel");
+                    } else {
+                        alert("Invalid email or password");
+                        throw new Error('Invalid email or password');
+                    }
+                })
+                .catch((error) => {
+                    console.error(error);
+                    // Show an error message to the user
+                });
+        }
+  
+    };
+
+
+
+
+
 
-function login() {
   return (
-    <Form>
-      <Form.Group className="mb-3" controlId="formBasicEmail">
-        <Form.Label>Email address</Form.Label>
-        <Form.Control type="email" placeholder="Enter email" />
-        <Form.Text className="text-muted">
-          We'll never share your email with anyone else.
-        </Form.Text>
-      </Form.Group>
 
-      <Form.Group className="mb-3" controlId="formBasicPassword">
+    
+    <form onSubmit={handleSubmit}>
+        <Form.Group className="mb-3" controlId="formBasicEmail">
+    <label>
+    <Form.Label>Email address</Form.Label>:
+
+    <Form.Control type="email" placeholder="Enter email" value={Email} onChange={(event) => setEmail(event.target.value)}/>
+
+    </label>
+    </Form.Group>
+    <label>
+    <Form.Group className="mb-3" controlId="formBasicPassword">
         <Form.Label>Password</Form.Label>
-        <Form.Control type="password" placeholder="Password" />
-      </Form.Group>
-      <Form.Group className="mb-3" controlId="formBasicCheckbox">
-        <Form.Check type="checkbox" label="Check me out" />
+
+        <Form.Control type="password" placeholder="Password" value={Password} onChange={(event) => setPassword(event.target.value)} />
+
+      
       </Form.Group>
-      <Button variant="primary" type="submit">
-        Submit
-      </Button>
-    </Form>
+    </label>
+    
+    <Button type="submit" variant="primary">
+        
+        Log in
+        
+    </Button>
+
+    <select id="options" value={selectedOption} onChange={handleOptionChange}>
+        <option value="0">Admin</option>
+        <option value="1">Dealer</option>
+      </select>
+
+  </form>
+
+    
+
   );
 }
 
-export default login;
\ No newline at end of file
+export default Login;
```

Comment:
when you see something like this it is just because you skip the new line in that file. Just add new line. You can avoid this type of warning.